src/log_migration.rb

```diff
@@ -44,3 +45,12 @@
         end
     end
 end
+time = Time.utc 2015,8,26
+LogTools::Converter.register "converter for DepthMap missing the time field", time, Orocos.registry do
+    conversion "/base/samples/DepthMap","/base/samples/DepthMap" do |dst,src|
+        deep_cast(dst,src,:self)
+        if not src.timestamps.empty? then
```

Comment:
Does this not only happen in case someone updates base/orogen/types and in the same time not updates base/types?